### PR TITLE
autogen: update english locales

### PR DESCRIPTION
**Pull request by zepatrik**
New messages were added to Kratos, and some wording changed on others.

The other languages will have to follow.

### CODE REVIEW

File: src/locales/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Ory Elements Default Translations
+
+This directory contains the default translations for Ory Elements. These
+translations are also used in the Ory Cloud Account Experience. Missing
+translations automatically fall back to the English version.
+
+## Automation
+
+The English base language file is auto-generated from various sources. Please
+update it by running `npm run generate-locales`.
```
Comment (jonas-jonas) [inline src/locales/README.md:10]: Should we make this a cronjob and create a PR with the result if there are changes?

**Review by jonas-jonas — APPROVED** [review]

**Comment by jonas-jonas** [comment]
The chromaui issue is fixed in https://github.com/ory/elements/pull/198

File: src/locales/README.md (hunk 1 above)
Comment (zepatrik) [reply src/locales/README.md:10]: Yeah, I was under the impression that there was such an action, but maybe I never came to do it.

**Comment by zepatrik** [comment]
Ok not sure why GH did not pick up that I pushed these commits to master? Usually that works 🤔

**Pull request closed without merging**